app/helpers/schema_helper.py

```diff
@@ -64,4 +64,10 @@ def is_goto_rule(rule):
 
     @staticmethod
     def is_goto_meta_rule(rule):
```

Comment:
I disagree, the rule uses metadata in it's evaluation always, despite how many conditions there are.